src/main/resources/db/migrationv2/V2_11__update_juror_add_response_entered_column.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE juror_mod.juror
+  ADD COLUMN response_entered BOOLEAN;
```

Comment:
Can we add a default of false here please

Reply:
updated